### ISSUE TITLE
AUR Archlinux package not available

### ISSUE THREAD

**Issue by kikislater** [DrTon/FlightPlot#61]
Hi,

On the main page of flightplot, I saw that there is an AUR package for Archlinux. The package is not available nowaday. https://aur.archlinux.org/packages/java-flightplot-git/
It's ok for me to run Flightplot with java -jar but the main page is not up to date.

**Comment by DrTon** [comment]
Hi! Honestly, I don't remember who added this, @ada-ci, @julianoes, any ideas?

**Comment by AndreasAntener** [comment]
Most likely @thomasgubler :)

**Comment by thomasgubler** [comment]
Yes that was me. The AUR was restructured some time ago. Someone ( @julianoes? ) would need to re-submit the pkgbuild in the new AUR git repo. PKGBUILD is here https://github.com/DrTon/FlightPlot/tree/master/packaging/archlinux

**Comment by julianoes** [comment]
@thomasgubler sorry, cant' help. Had to switch to Ubuntu.

**Comment by kikislater** [comment]
I tried to build it and found a problem. 

> $ makepkg 
> ==> Création du paquet java-flightplot-git 20150101-1 (lun. avril 11 11:41:21 MUT 2016)
> ==> Vérification des dépendances pour l’exécution...
> ==> Vérification des dépendances pour la compilation...
> ==> Récupération des sources...
>   -> Clonage du dépôt FlightPlot git...
> Clonage dans le dépôt nu '/tmp/aurtmp/FlightPlot'
> remote: Counting objects: 2643, done.
> remote: Total 2643 (delta 0), reused 0 (delta 0), pack-reused 2643
> Réception d'objets: 100% (2643/2643), 3.40 MiB | 35.00 KiB/s, fait.
> Résolution des deltas: 100% (1301/1301), fait.
> Vérification de la connectivité... fait.
> ==> Validation des fichiers source avec md5sums...
>     FlightPlot ... Ignoré
> ==> Extraction des sources...
>   -> Création d’une copie de travail du dépot FlightPlot git...
> Clonage dans 'FlightPlot'...
> fait.
> ==> Lancement de prepare()...
> Sous-module 'jMAVlib' (https://github.com/DrTon/jMAVlib.git) enregistré pour le chemin 'jMAVlib'
> Clonage dans 'jMAVlib'...
> remote: Counting objects: 557, done.
> remote: Total 557 (delta 0), reused 0 (delta 0), pack-reused 557
> Réception d'objets: 100% (557/557), 92.12 KiB | 74.00 KiB/s, fait.
> Résolution des deltas: 100% (215/215), fait.
> Vérification de la connectivité... fait.
> Chemin de sous-module 'jMAVlib' : '1d2af0b9ea9738a570cea99f31038aac23b8ebad' extrait
> ==> Lancement de pkgver()...
> /tmp/aurtmp/PKGBUILD: ligne 42 : cd: java-flightplot-git: Aucun fichier ou dossier de ce type
> ==> ERREUR : pkgver ne peut pas être vide.
> ==> ERREUR : pkgver() a généré une version non valide : 

I create a directory and now works
